CollAction/Migrations/20190407192822_CA_591_remove_category_friesland.cs

```diff
@@ -0,0 +1,17 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace CollAction.Migrations
+{
+    public partial class CA_591_remove_category_friesland : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.Sql("DELETE FROM \"Categories\" WHERE \"Name\" = 'Friesland'");
+        }
+
+        protected override void Down(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.Sql("DELETE FROM \"Categories\" WHERE \"Name\" = 'Friesland'");
```

Comment:
On "Down" it needs to recreate the category. The "Down" action is supposed to reverse the migration, so we can always roll back database actions.

Reply:
I understand that, but do you want to create an update/insert with that data ? I have been following similiar issue with update that was refered, which does the same for Up/Down

Reply:
For some reason I do not have that entry in my local DB from master, so I do not know how to set the color column for it 🤦‍♂️ as it is stored as an integer.

Reply:
You can check the Friesland branch to see which data you'd need to re-insert. The list of categories and the exact data can be found in the function "CreateCategories" in ApplicationDbContext.cs.

Although, in this case, it's probably good enough to keep it empty. Doubt we'd need to roll this.

Reply:
Added Down insert with default color in ebbee14